tests/fixtures/agent.py

```diff
@@ -42,3 +43,23 @@ def reflexion_react_agent() -> ReflexionReActAgent:
         action_llm=FakeListChatModel(responses=["1"]),
     )
     return agent
+
+
+@pytest.fixture
+def alfworld_env(alfworld_file):
+    """Prepare for env init for Alfworld."""
+    with open(alfworld_file) as reader:
+        config = yaml.safe_load(reader) 
+    return config
+
+
+@pytest.fixture
+def data_dir(pytestconfig):
+    """Dir path to sleap data."""
+    return Path(pytestconfig.rootdir) / "tests/assets"
+
+
+@pytest.fixture
+def alfworld_file(data_dir):
+    """Sleap single fly .slp and video file paths."""
```

Comment:
order the functions from top to bottom:

data_dir
alfworld_file
alfworld_env